src/App.js

```diff
@@ -1,39 +1,62 @@
 //TODO: STEP 1 - Import the useState hook.
-import React from "react";
+import React, {useState} from "react";
 import "./App.css";
 import BottomRow from "./BottomRow";
 
 function App() {
   //TODO: STEP 2 - Establish your applictaion's state with some useState hooks.  You'll need one for the home score and another for the away score.
-
+  const [homeScore, setHomeScore ] = useState(0);
+  const [awayScore, setAwayScore ] = useState(0);
+  const touchdownHome = e => {
+    setHomeScore(homeScore + 7)
+  }
+  const touchdownAway = e => {
+    setAwayScore(awayScore + 7)
+  }
+  const fieldHome = e => {
+    setHomeScore(homeScore + 3)
+  }
+  const fieldAway = e => {
+    setAwayScore(awayScore + 3)
+  }
+  const [quarter, setQuarter] = useState(0)
+  const nextQuarter = e => {
+    if (quarter < 4){
+      setQuarter(quarter + 1)
+    }
+    else { setQuarter("Game over")}
+    
+  }
```

Comment:
Nice work on organizing your code to make it more readable Josh !! 

While this project was rather small in way of changes I would like your commits to be more descriptive. Think like your providing a company with an important update to the project. Use your commits like an instruction set for example
'Implement scoreboard functionality' A commit not only provides your peers information on changes it also provides you with a reference point that if needed would allow you to revert your code back to a certain point.